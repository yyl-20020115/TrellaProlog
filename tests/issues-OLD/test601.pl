main :-
	L=['3'|L],
	Es=['0'|L],
	writeq(L), nl,
	writeq(Es), nl.

:- initialization(main).
