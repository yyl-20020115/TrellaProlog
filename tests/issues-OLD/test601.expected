['3'|L]
['0','3'|L]
